### PR TITLE
Update documentation on Elastic Agent statuses

### PR DESCRIPTION
**Pull request by mdbirnstiehl**
Closes [Issue 2753](https://github.com/elastic/observability-docs/issues/2753).

Agent statuses to show current processes. I also added a diagram to show the flow of the different statuses.

[Preview of the docs](https://observability-docs_2822.docs-preview.app.elstc.co/guide/en/fleet/master/monitor-elastic-agent.html#view-agent-status)

### CODE REVIEW

**Comment by mdbirnstiehl** [comment]
@nimarezainia I updated the agent statuses and added the diagram to match what's in your [google doc](https://docs.google.com/document/d/1sWXzljzvuIccpreHou-xD7STo6nSpy6vWhDVzdVI6jw/edit?usp=sharing). Let me know if this looks ok or if you see any needed changes.

**Review by nimarezainia — APPROVED** [review]

File: docs/en/ingest-management/fleet/monitor-elastic-agent.asciidoc [hunk 1]
```diff
@@ -23,26 +23,31 @@ To view the status of your {fleet}-managed agents, in {kib}, go to
...
+| *Healthy* | Healthy {agent}s are enrolled and checked-in. There are no agent policy updates or automatic agent binary updates in progress, but the agent binary may still be out of date. {agent}s continuously check in to the {fleet-server} 
+for required updates. 
 
-| **Unhealthy** | The {agent} is online, but requires attention because it's
-reporting either agent or integration-level errors.
+| *Unhealthy* | Unhealthy {agent}s have not checked in to {fleet-server}. At this point, you may need to address the situation. 
 
-| **Updating** | The {agent} is online and updating the agent policy, updating the
-binary, or enrolling or unenrolling from {fleet}.
+| *Updating* | Updating {agent}s are updating the agent policy, updating the binary, or enrolling or unenrolling from Fleet.
```
Comment (kilfoyle) [inline docs/en/ingest-management/fleet/monitor-elastic-agent.asciidoc:35]:
```suggestion
| *Updating* | Updating {agent}s are updating the agent policy, updating the binary, or enrolling or unenrolling from {fleet}.
```

File: docs/en/ingest-management/fleet/monitor-elastic-agent.asciidoc [hunk 2]
```diff
@@ -23,26 +23,31 @@ To view the status of your {fleet}-managed agents, in {kib}, go to
...
 
-{fleet} reports these statuses:
+{agent}s can have the following statuses:
 
 |===
 
-| **Healthy** | The {agent} is online and running as expected. There are no agent
-policy updates or automatic agent binary updates in progress, but the
-agent binary may still be out of date.
+| *Healthy* | Healthy {agent}s are enrolled and checked-in. There are no agent policy updates or automatic agent binary updates in progress, but the agent binary may still be out of date. {agent}s continuously check in to the {fleet-server}
```
Comment (kilfoyle) [inline docs/en/ingest-management/fleet/monitor-elastic-agent.asciidoc:30]:
```suggestion
| *Healthy* | Healthy {agent}s are enrolled and checked in. There are no agent policy updates or automatic agent binary updates in progress, but the agent binary may still be out of date. {agent}s continuously check in to {fleet-server}.
```

File: docs/en/ingest-management/fleet/monitor-elastic-agent.asciidoc [hunk 3]
```diff
@@ -23,26 +23,31 @@ To view the status of your {fleet}-managed agents, in {kib}, go to
...
 
-| **Updating** | The {agent} is online and updating the agent policy, updating the
-binary, or enrolling or unenrolling from {fleet}.
+| *Updating* | Updating {agent}s are updating the agent policy, updating the binary, or enrolling or unenrolling from Fleet.
 
-| **Offline** | The {agent} has not checked in during the expected time period.
+| *Offline* | Offline {agent}s have stayed in an unhealthy status for a period of time. Offline agent's API keys remain valid. You can still see these {agent}s in the {fleet} UI and investigate them for further diagnosis if required.
 
-| **Inactive** | The {agent} is valid, but has been removed from the main {fleet} UI according to your <<set-inactivity-timeout,inactivity timeout>>.
+| *Inactive* | Inactive {agent}s have been offline for longer than the time set in your <<set-inactivity-timeout,inactivity timeout>>. These {agent}s are valid, but have been removed from the main {fleet} UI.
```
Comment (kilfoyle) [inline docs/en/ingest-management/fleet/monitor-elastic-agent.asciidoc:39]:
```suggestion
| *Inactive* | {agent}s have been offline for longer than the time set in your <<set-inactivity-timeout,inactivity timeout>>. These {agent}s are valid, but have been removed from the main {fleet} UI. 
```
For this and each of the other rows, is the first word meant to be repeated (i.e., `Inactive`, `Unenrolled`, `Updating`, `Healthy`, and `Unhealthy`? It looks to me like these terms shouldn't be repeated, but maybe I misunderstand something.

Screenshot 2023-04-10 at 1 50 27 PM
--
Screenshot 2023-04-10 at 1 51 46 PM

**Review by kilfoyle — APPROVED** [review]
LGTM! 🎸

**Pull request merged**